alacritty/src/display/mod.rs

```diff
@@ -568,31 +582,31 @@ impl Display {
         let (mut cell_width, mut cell_height) =
             (self.size_info.cell_width(), self.size_info.cell_height());
 
-        // Ensure we're modifying the correct OpenGL context.
-        self.window.make_current();
+        if pending_update.font().is_some() || pending_update.cursor_dirty() {
+            let pending_renderer_update =
+                self.pending_renderer_update.get_or_insert(Default::default());
+            pending_renderer_update.clear_font_cache = true
```

Comment:
```suggestion
            let renderer_update = self.pending_renderer_update.get_or_insert(Default::default());
            renderer_update.clear_font_cache = true
```

Just a super short temporary variable, should be fine to use a little bit of a shorthand.